test/test_cfmodel.py

```diff
@@ -0,0 +1,21 @@
+from carla.cf_models.catalog.dice import Dice
+from carla.data.catalog import DataCatalog
+from carla.models.catalog import MLModelCatalog
+from carla.models.negative_instances import predict_negative_instances
+
+
+def test_dice_get_counterfactuals():
+    # Build data and mlmodel
+    data_name = "adult"
+    data_catalog = "adult_catalog.yaml"
+    data = DataCatalog(data_name, data_catalog, drop_first_encoding=True)
+    model_tf = MLModelCatalog(data, data_name, "ann")
+    # get factuals
+    factuals = predict_negative_instances(model_tf, data)
```

Comment:
I actually like the `factuals` naming. If thats common terminology should we rename `predict_negative_instaces` to `predict_factuals`?

Reply:
Maybe `predic_negative_factuals` to state that this method chooses always negative predicted factuals?

Reply:
ah alright, factuals are all the predictions then. Yeah no strong opinions, also fine to leave it like it is now 👍🏽